toolkit/tools/imagecustomizerapi/user.go

```diff
@@ -17,6 +17,7 @@ type User struct {
 	PasswordPath        string   `yaml:"PasswordPath"`
 	PasswordExpiresDays *int64   `yaml:"PasswordExpiresDays"`
 	SSHPubKeyPaths      []string `yaml:"SSHPubKeyPaths"`
+	SSHPubKeys          []string `yaml:"SSHPubKeys"`
```

Comment:
Adding this should probably be done in a separate change.

Reply:
ok will include it in a separate pr